### ISSUE TITLE
[Bug]: Netflix app stuck on loading spinner

### ISSUE THREAD

**Issue by larryqiann** [PlayCover/PlayCover#1761]
### Describe the bug

Using macOS 15.1.1 and PlayCover 3.1.0 (856) and version 17.7.1 of the Netflix app from both the recommended sources, the app opens but after the Netflix loading animation, displays a loading spinner only.

### Steps to reproduce

Using PlayCover 3.1.0 (856) [edit: same issue using nightly 952] and version 17.7.1 of the Netflix app from both the recommended sources, install the app and note that the app opens but after the Netflix loading animation, displays a loading spinner only. A box to allow Netflix to access data from other apps appears after a while or when closing the app, but does not do anything when allowed.

### Expected behaviour

The app should show the login screen

### What version of PlayCover are you using?

3.1.0

### What version of macOS are you using?

Other (please specify)

### Issue Language

- [X] Yes my issue is written in English

**Comment by larryqiann** [comment]
![CleanShot 2024-12-07 at 01 17 52](https://github.com/user-attachments/assets/3508ef38-bc7b-4941-88cb-3316a9d24451)

**Comment by Aaron1238** [comment]
Same, idk how to fix this. Which store did you download the ipa from?

**Comment by larryqiann** [comment]
I tried both from the Wiki.

**Comment by hershdhami** [comment]
Same and I have no clue how to fix this - I downloaded from the sites listed in this [article](https://machow2.com/netflix-on-mac/)

**Comment by CodexJeff** [comment]
Same issue here

**Comment by paKS7** [comment]
Same problem here, since upgrading to MacOS 15.1.1 Sequoia. Used to use Netflix 16.12, which crashes on start over PlayCover since the OS update. Tried several combinations of Netflix apps and PlayCover versions:
PlayCover 3.1.0 with Netflix 17.7.x 17.9.x show the above mentioned infinite loading screen with a popup saying "com.netflix.Netflix wants to access your local data" (which doesn't make so much sense for a sandbox app). Hope there is a fix out there!

**Comment by johnz45** [comment]
Yeah, the same. Spinning indicator while running sequoia and latest versions of both apps.

**Comment by gmontard** [comment]
It seems it's not solved :( 
Anyone trying to make it work?

**Comment by vinimaz** [comment]
Netflix makes it on purpose using a check in the code with `isMacCatalystApp `, there is one way to modify the IPA searching for this part and modifying it. It is not my patch neither my code, but it works.
Here https://gist.github.com/bensaine/f675dbde9fdd60d4671651c378838000
